app/jobs/asset_cleanup_job.rb

```diff
@@ -37,10 +37,12 @@ def clean_file_attachment_assets
   end
 
   def delete_asset(asset)
-    GdsApi.asset_manager.delete_asset(asset.asset_manager_id)
-  rescue GdsApi::HTTPNotFound
-    logger.warn("No asset to delete for id #{asset.asset_manager_id}")
-  ensure
+    begin
+      GdsApi.asset_manager.delete_asset(asset.asset_manager_id)
+    rescue GdsApi::HTTPNotFound
+      logger.warn("No asset to delete for id #{asset.asset_manager_id}")
+    end
+
```

Comment:
Nitpick, I think commit message is missing a comma after "ensure".

Reply:
Thank you - I've reworded the commit and will merge when it's green.

Reply:
The commit message is much better! Thanks!